### DISCUSSION TITLE
Is there a way to disable Drive Database auto updates breaking drive compatibility?

### DISCUSSION THREAD

**Discussion by Gadgetblues** [007revad/Synology_HDD_db#266] [Q&A] [answered]
Every time I get a Drive Database update it breaks my drive compatibility, tons of warnings fire off, and I have to manually re-run the hdd_db script. This happens pretty often, e.g. Feb 1, Feb 21, Mar 1, Mar 21.

I know the script is supposed to disable updates but it isn't working.

Is there any way to disable Drive Database updates? Or somehow trigger the script automatically when it happens?

**Comment by 007revad** [comment]
DSM will check for Drive Database updates when DSM updates or you insert a new drive. There are other things that trigger a check for Drive Database updates, but I'm not sure what they are. 

> I know the script is supposed to disable updates but it isn't working.
1. Do you run the script with the -n or --noupdate option?
2. Do you have the script scheduled to run at boot up?

**Comment by Gadgetblues** [comment]
I need to specify that? The docs say "Prevents DSM auto updating the drive database" not "Optionally Prevents DSM auto updating the drive database" so I assumed it was the default. Docerror? I will change the command line.

**Comment by 007revad** [comment, marked as answer]
That's embarrassing.

12 months ago, in v1.2.25 I changed it to be optional... but forgot to edit the readme to say it's now optional. I've fixed the readme now.

**Comment by Gadgetblues** [comment]
Awesome, thank you!
